project03.py

```diff
@@ -232,7 +232,12 @@ def verifyDeathBefore150YearsOld(person):
         age = death_unix - bday_unix
     years_in_seconds = 150 * 365 * 24 * 60 * 60
     return age <= years_in_seconds
-
+def noBigamy(person):
+    #if the person does not have a spouse
```

Comment:
This isn't bigamy, bigamy is when marriages double up. For example, when person A is married to B but person C also says they are married B. No matter what B is married to on the gedcom file, this is bigamy and shouldn't happen.